modules/apps/staging/staging-taglib/src/main/resources/META-INF/resources/portlet_list/page.jsp

```diff
@@ -88,14 +88,27 @@ StagingGroupHelper stagingGroupHelper = StagingGroupHelperUtil.getStagingGroupHe
 	%>
 
 		<li class="tree-item <%= ((exportModelCount > 0) || showAllPortlets) ? StringPool.BLANK : "deletions" %>">
-			<liferay-staging:checkbox
-				checked="<%= showPortletDataInput %>"
-				deletions="<%= modelDeletionCount %>"
-				disabled="<%= disableInputs %>"
-				items="<%= exportModelCount %>"
-				label="<%= portletTitle %>"
-				name="<%= PortletDataHandlerKeys.PORTLET_DATA + StringPool.UNDERLINE + portlet.getPortletId() %>"
-			/>
+			<c:choose>
+				<c:when test="<%= portletDataHandler.isModelCountSupported() %>">
```

Comment:
As we discussed, I think we need a different approach:

- Setting `exportModelCount` to `0` always if `isModelCountSupported` is false
- Including condition in the if clause to continue if not displayCounts and not isModelCountSupported